conf/logrotate.conf

```diff
@@ -20,6 +20,23 @@ notifempty
   endscript
 }
 
+## AUTH
+/var/log/st2/st2auth.log {
```

Comment:
Should this be "/var/log/st2/st2auth*.log" since the audit entry added below is identical to this one?